### PR TITLE
Update Solarized Dark Theme

### LINKED ISSUE #6: Solarized Dark cshtml colors look wrong
I found a problem with Solarized Dark theme in cshtml files.
The C# code is highlighed in gray.

img

### PR DESCRIPTION
**Pull request by styfle**
I followed @prnadago steps to generate XML file from the VS Theme.
This should hopefully fix the razor syntax highlighting as demonstrated [here](https://stackoverflow.com/a/28404477/266535).

Fixes #6

### CODE REVIEW

**Comment by SalahAdDin** [comment]
Yeah, actually, we cannot see the letters in black, there is not a good contrast there, can you also review it?

File: VSColorThemes/Themes/SolarizedDark.xml [hunk 1]
```diff
@@ -120,8 +326,27 @@
     </Category>
     <Category Name="Autos" GUID="{a7ee6bee-d0aa-4b2f-ad9d-748276a725f6}">
       <Color Name="ChangedText">
+        <Background Type="CT_INVALID" Source="00000000" />
```
Comment (Newrad0603) [inline VSColorThemes/Themes/SolarizedDark.xml]: Please remove all instances of CT_INVALID color entries. They are non-functional, but bloat the private registry and make diffing real changes harder.

File: VSColorThemes/Themes/SolarizedDark.xml [hunk 2]
```diff
@@ -120,8 +326,27 @@
...
         <Foreground Type="CT_RAW" Source="FFF7A1A1" />
       </Color>
+      <Color Name="Plain Text">
```
Comment (Newrad0603) [inline VSColorThemes/Themes/SolarizedDark.xml]: Once the CT_INVALID entries are removed, you'll end up with empty colors, so make sure to remove those as well. Once the empty colors are removed, you may have empty categories which should also be removed.

**Comment by Newrad0603** [comment]
Can you provide screenshots of the before/after colors?

**Comment by styfle** [comment]
No I cannot, because I use macOS now

**Pull request closed without merging**